src/Protocol/ActivityPub/Processor.php

```diff
@@ -856,7 +856,7 @@ private static function acceptIncomingMessage(array $activity, string $id)
 			}
 			// We check with "strpos" for performance issues. Only when this is true, the regular expression check is used
 			// RegExp is taken from here: https://medium.com/@shiba1014/regex-word-boundaries-with-unicode-207794f6e7ed
-			if ((strpos($content, $tag) !== false) && preg_match('/(?<=[\s,.:;"\']|^)' . $tag . '(?=[\s,.:;"\']|$)/', $content)) {
+			if ((strpos($content, $tag) !== false) && preg_match('/(?<=[\s,.:;"\']|^)' . preg_quote($tag) . '(?=[\s,.:;"\']|$)/', $content)) {
```

Comment:
Halfway there, you also need to provide the expression delimiter character as a second argument so that it can be escaped as well, in this case it is `/`.

Reply:
Done